API.md

```diff
@@ -1216,11 +1216,14 @@ can call:
     Cleans up any event handlers Flot has currently registered. This
     is used internally.
 
- - findNearbyItem(mouseX, mouseY, seriesFilter, distance)
+ - findNearbyItem(mouseX, mouseY, seriesFilter, distance, distanceFunction)
```

Comment:
it would be better if you make it:

findNearbyItem(mouseX, mouseY, seriesFilter, radius, computeDistance)